components/account-details.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="account-details">
+    <h3 class="h4">
+      Account Details
+    </h3>
+    <dl v-if="user" class="account-details__list">
+      <dt>Name</dt>
+      <dd>{{ name }}</dd>
+      <dt>Email address</dt>
+      <dd>{{ email }}</dd>
+      <dt>Phone number</dt>
+      <dd>{{ phone }}</dd>
+    </dl>
+    <div v-else class="account-details__placeholder">
+      <p>Please login to view your account details</p>
+    </div>
+    <div class="account-details__buttons">
+      <UiButton v-if="!user" kind="primary" @click="login">
+        Log in
```

Comment:
login, logout?
I know it is correct as verb too, but sounds weird as a button.

Reply:
Done